support.html

```diff
@@ -0,0 +1,54 @@
+<!--#include virtual="includes/_header.htm" -->
+<body class="page-contact ">
+<!--#include virtual="includes/_top.htm" -->
+<div class="content">
+	<!--#include virtual="includes/_nav.htm" -->
+	<div class="right">
+		<h1 class="content-title">Support</h1>
+		<h3>Community support</h3>
+
+		<p>
+			This is an open source project so the amount of time community members have available to help resolve your issue
+			is often limited as help is provided on a volunteer basis from
+			<a href="https://www.apache.org/foundation/how-it-works/#hats" target="_blank">individuals</a>.
+			However, it is free, and you are often able to discuss issues with the developers who actually wrote the code
+			you are running.
+		</p>
+		<p>
+			If you want community support then please <a href="https://kafka.apache.org/contact">contact us</a>.
+			If you’re fairly certain you’re hitting a bug please report it via one of our
+			<a href="https://kafka.apache.org/contributing">issue trackers</a>. Be sure to review
+			these pages carefully as they contain tips and tricks about working with Apache communities in general and
+			Apache Kafka in particular.
+		</p>
+
+		<h3>Commercial support</h3>
+		<p>
+			Apache Kafka is a widely used project. As such, several companies have built products and services around Kafka.
+			This page is dedicated to providing descriptions of those offerings and links to more information.
+			Companies are definitely encouraged to update this page directly or send a mail to the Kafka PMC with a description
+			of your offerings, and we can update the page. The products and services listed on this page are provided for
+			information use only to our users. The Kafka PMC does not endorse or recommend any of the products or services
+			on this page. See below for information about what is appropriate to add to the page.
+		</p>
+
+		<ul>
+			<li><a href="https://www.yupiik.com" target="_blank">Yupiik</a> contributes and commits to many Apache projects. Provides consulting, training and support
```

Comment:
It seems very weird to me to add this page with only a single commercial support offering listed. Honestly it feels a bit inappropriate to use the AK site to list any commercial offerings at all, but I'm not sure what the official policy is there. If we do include a commercial support section then imo we should consider reaching out to some of the other major companies offering Kafka support to ask if they would like to be included so that we provide a more complete picture of the available options. Otherwise it feels like we (the Apache Kafka project) are advocating for a particular commercial offering, which we definitely are not and should not.

I still think the first part of this page could be very valuable (with the above suggestions) and would recommend splitting this PR up so that we can merge the community support section while discussing the suitability of the commercial support section

Reply:
An alternative to reaching out (which feels like the opposite) is to add an edit link (github) and encourage people to pr to add themselves.
Will make it more open and less closed to a single company so cna be a trade off maybe.
Wdyt?

Reply:
I added the first company because I don't know the other and like it's mention in the text, company are encouraged to add themselves by providing a PR. At the beginning we need to start with one company but if people want to add other companies in this PR with comments, I can add them to start with more than one before merging.

Reply:
@ableegoldman do you think we can start a thread on the mailing list to ask which company want to be add in the commercial list support?

Reply:
I think you should start a discussion thread to ask if we "should" have the "Commercial support" section first. I agree with @ableegoldman that it's weird the Apache Kafka site contains any commercial content.